packages/components/inputs/localized-text-input/src/localized-text-input.styles.ts

```diff
@@ -17,7 +19,10 @@ const getLocalizedInputStyles = () => [
   `,
 ];
 
-const getLanguageLabelStyles = (props, theme) => {
+const getLanguageLabelStyles = (
+  _props: LocalizedTextInputStylesProps,
```

Comment:
I'm wondering if we could use the `unknown` type instead of creating an empty one 🤔